src/cloudinary.js

```diff
@@ -652,7 +654,7 @@ class Cloudinary {
     }
 
     let roundDpr = options.round_dpr != null ? options.round_dpr : this.config('round_dpr');
-    elements.forEach(tag => {
+    elements.forEach((tag) => {
```

Comment:
don't need `()` here

Reply:
Braces are required because of `arrow-body-style`.